dbflute-engine/src/main/java/org/dbflute/logic/doc/spolicy/DfSPolicyColumnStatementChecker.java

```diff
@@ -205,6 +213,17 @@ protected void evaluateColumnThenTheme(DfSPolicyStatement statement, DfSPolicyRe
             if (!column.hasClassification() == !notThenClause) {
                 result.violate(policy, "The column should " + notOr + "be classification: " + toColumnDisp(column));
             }
+        } else if (Srl.startsWithIgnoreCase(thenTheme, "classification")) {
+            Matcher matcher = clsPattern.matcher(thenTheme);
+            if (matcher.find()) {   // surely go into if statement
```

Comment:
Does the if-clause match with simple pattern "then classification" ?

Reply:
The code, `mactcher.find()` does not match simple pattern 'then classification'.
I think that it does not need to match, because the previous else if statement should check the simple pattern.

Reply:
So, I am wondering to use Matcher of "classification\((.*?)\)" again instead of `Srl.startsWithIgnoreCase(thenTheme, "classification"`.